data/inputs/gri30_ion.cti

```diff
@@ -2,22 +2,157 @@ units(length='cm', time='s', quantity='mol', act_energy='cal/mol')
 
 ideal_gas(name='gas',
           elements=' O  H  C  N  Ar  E',
-          species=['''gri30: H2  H  O  O2  OH  H2O  HO2  H2O2  C  CH 
-                   CH2  CH2(S)  CH3  CH4  CO  CO2  HCO  CH2O  CH2OH  CH3O 
-                   CH3OH  C2H  C2H2  C2H3  C2H4  C2H5  C2H6  HCCO  CH2CO  HCCOH 
-                   N  NH  NH2  NH3  NNH  NO  NO2  N2O  HNO  CN 
-                   HCN  H2CN  HCNN  HCNO  HOCN  HNCO  NCO  N2  AR  C3H7 
+          species=['H2  O2  H2O  CH4  CO  CO2  N2',
+                   '''gri30: H  O  OH  HO2  H2O2  C  CH
+                   CH2  CH2(S)  CH3  HCO  CH2O  CH2OH  CH3O
+                   CH3OH  C2H  C2H2  C2H3  C2H4  C2H5  C2H6  HCCO  CH2CO  HCCOH
+                   N  NH  NH2  NH3  NNH  NO  NO2  N2O  HNO  CN
+                   HCN  H2CN  HCNN  HCNO  HOCN  HNCO  NCO  AR  C3H7
                    C3H8  CH2CHO  CH3CHO''',
                    'HCO+    H3O+    E'],
           reactions=['gri30: all', 'all'],
-          transport='Mix',
+          transport='Ion',
           options=['skip_undeclared_species', 'skip_undeclared_third_bodies'],
           initial_state=state(temperature=300.0, pressure=OneAtm))
 
 #-------------------------------------------------------------------------------
 # Species data
 #-------------------------------------------------------------------------------
 
+species(name = "H2",
+    atoms = " H:2 ",
+    thermo = (
+       NASA( [  200.00,  1000.00], [  2.344331120E+00,   7.980520750E-03, 
+               -1.947815100E-05,   2.015720940E-08,  -7.376117610E-12,
+               -9.179351730E+02,   6.830102380E-01] ),
+       NASA( [ 1000.00,  3500.00], [  3.337279200E+00,  -4.940247310E-05, 
+                4.994567780E-07,  -1.795663940E-10,   2.002553760E-14,
+               -9.501589220E+02,  -3.205023310E+00] )
+             ),
+    transport = gas_transport(
+                     geom = "linear",
+                     diam =     2.92,
+                     well_depth =    38.00,
+                     polar =     0.455,
+                     rot_relax =   280.00),
+    note = "TPIS78"
```

Comment:
For those species where the transport coefficients are different from `gri30.cti`, I think it needs to be documented where those changes came from.

For those species where there aren't any changes to the coefficients, those species' definitions shouldn't be copied into this file.